packages/ui-components/src/components/Icons.tsx

```diff
@@ -1757,6 +1796,22 @@ export function initIcons(): void {
                 </svg>
             ),
             [UiIcons.Source]: (
+                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M0 9.5C0.456153 9.68246 0.871447 9.86492 1.25198 10.0535C3.03994 10.9394 4.06055 11.9601 4.94651 13.748C5.13508 14.1286 5.31754 14.5438 5.5 15C5.68246 14.5438 5.86493 14.1286 6.05349 13.748C6.93945 11.9601 7.96006 10.9394 9.74802 10.0535C10.1286 9.86493 10.5438 9.68246 11 9.5C10.5438 9.31754 10.1286 9.13507 9.74802 8.94651C7.96006 8.06055 6.93945 7.03994 6.05349 5.25198C5.86493 4.87145 5.68246 4.45615 5.5 4C5.31754 4.45615 5.13508 4.87145 4.94651 5.25198C4.06055 7.03994 3.03994 8.06055 1.25198 8.94651C0.871447 9.13508 0.456153 9.31754 0 9.5ZM2.3404 9.5C3.02452 9.89132 3.61849 10.3293 4.14461 10.8554C4.67072 11.3815 5.10868 11.9755 5.5 12.6596C5.89132 11.9755 6.32928 11.3815 6.85539 10.8554C7.38151 10.3293 7.97548 9.89132 8.6596 9.5C7.97548 9.10868 7.38151 8.67072 6.85539 8.14461C6.32928 7.61849 5.89132 7.02452 5.5 6.3404C5.10868 7.02452 4.67072 7.61849 4.14461 8.14461C3.61849 8.67072 3.02452 9.10868 2.3404 9.5Z"
+                        fill={COLORS.default}
+                    />
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M9 3.5C9.45862 3.68345 9.85233 3.86689 10.1965 4.06565C10.9672 4.51082 11.4892 5.03278 11.9344 5.80355C12.1331 6.14767 12.3166 6.54138 12.5 7C12.6834 6.54138 12.8669 6.14767 13.0656 5.80355C13.5108 5.03278 14.0328 4.51082 14.8035 4.06565C15.1477 3.86689 15.5414 3.68345 16 3.5C15.5414 3.31655 15.1477 3.13311 14.8036 2.93435C14.0328 2.48918 13.5108 1.96722 13.0656 1.19645C12.8669 0.85233 12.6834 0.458615 12.5 0C12.3166 0.458615 12.1331 0.852329 11.9344 1.19645C11.4892 1.96722 10.9672 2.48918 10.1965 2.93435C9.85233 3.13311 9.45862 3.31655 9 3.5ZM11.1697 3.5C11.4296 3.68174 11.6705 3.88127 11.8946 4.10539C12.1187 4.32952 12.3183 4.57043 12.5 4.83033C12.6817 4.57043 12.8813 4.32952 13.1054 4.10539C13.3295 3.88127 13.5704 3.68173 13.8303 3.5C13.5704 3.31826 13.3295 3.11873 13.1054 2.89461C12.8813 2.67048 12.6817 2.42957 12.5 2.16967C12.3183 2.42957 12.1187 2.67048 11.8946 2.89461C11.6705 3.11873 11.4296 3.31826 11.1697 3.5Z"
+                        fill={COLORS.default}
+                    />
+                </svg>
+            ),
+            [UiIcons.Sparkles]: (
```

Comment:
I think `UiIcons.Source` and `UiIcons.Sparkles` have the svg content the wrong way around here?

Reply:
Yes, you're right. These got mixed up somehow.
          
`UiIcons.Source` should be:

`[UiIcons.Source]: (
                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
                    <path
                        fill={COLORS.default}
                        fillRule="evenodd"
                        d="M5.24525,3 L5.99525,3.662 L2.16725,7.997 L5.99925,12.338 L5.24925,13 L1.12525,8.328 C0.979125,8.162625 0.960859375,7.92451562 1.07045312,7.74095703 L1.12525,7.666 L5.24525,3 Z M10.754,3 L14.874,7.666 C15.041,7.855 15.041,8.139 14.874,8.328 L10.75,13 L10,12.338 L13.832,7.997 L10.004,3.662 L10.754,3 Z"
                    />
                </svg>
            ),`  

`UiIcons.Sparkles` should be:

` [UiIcons.Source]: (
                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
                    <path
                        fillRule="evenodd"
                        clipRule="evenodd"
                        d="M0 9.5C0.456153 9.68246 0.871447 9.86492 1.25198 10.0535C3.03994 10.9394 4.06055 11.9601 4.94651 13.748C5.13508 14.1286 5.31754 14.5438 5.5 15C5.68246 14.5438 5.86493 14.1286 6.05349 13.748C6.93945 11.9601 7.96006 10.9394 9.74802 10.0535C10.1286 9.86493 10.5438 9.68246 11 9.5C10.5438 9.31754 10.1286 9.13507 9.74802 8.94651C7.96006 8.06055 6.93945 7.03994 6.05349 5.25198C5.86493 4.87145 5.68246 4.45615 5.5 4C5.31754 4.45615 5.13508 4.87145 4.94651 5.25198C4.06055 7.03994 3.03994 8.06055 1.25198 8.94651C0.871447 9.13508 0.456153 9.31754 0 9.5ZM2.3404 9.5C3.02452 9.89132 3.61849 10.3293 4.14461 10.8554C4.67072 11.3815 5.10868 11.9755 5.5 12.6596C5.89132 11.9755 6.32928 11.3815 6.85539 10.8554C7.38151 10.3293 7.97548 9.89132 8.6596 9.5C7.97548 9.10868 7.38151 8.67072 6.85539 8.14461C6.32928 7.61849 5.89132 7.02452 5.5 6.3404C5.10868 7.02452 4.67072 7.61849 4.14461 8.14461C3.61849 8.67072 3.02452 9.10868 2.3404 9.5Z"
                        fill={COLORS.default}
                    />
                    <path
                        fillRule="evenodd"
                        clipRule="evenodd"
                        d="M9 3.5C9.45862 3.68345 9.85233 3.86689 10.1965 4.06565C10.9672 4.51082 11.4892 5.03278 11.9344 5.80355C12.1331 6.14767 12.3166 6.54138 12.5 7C12.6834 6.54138 12.8669 6.14767 13.0656 5.80355C13.5108 5.03278 14.0328 4.51082 14.8035 4.06565C15.1477 3.86689 15.5414 3.68345 16 3.5C15.5414 3.31655 15.1477 3.13311 14.8036 2.93435C14.0328 2.48918 13.5108 1.96722 13.0656 1.19645C12.8669 0.85233 12.6834 0.458615 12.5 0C12.3166 0.458615 12.1331 0.852329 11.9344 1.19645C11.4892 1.96722 10.9672 2.48918 10.1965 2.93435C9.85233 3.13311 9.45862 3.31655 9 3.5ZM11.1697 3.5C11.4296 3.68174 11.6705 3.88127 11.8946 4.10539C12.1187 4.32952 12.3183 4.57043 12.5 4.83033C12.6817 4.57043 12.8813 4.32952 13.1054 4.10539C13.3295 3.88127 13.5704 3.68173 13.8303 3.5C13.5704 3.31826 13.3295 3.11873 13.1054 2.89461C12.8813 2.67048 12.6817 2.42957 12.5 2.16967C12.3183 2.42957 12.1187 2.67048 11.8946 2.89461C11.6705 3.11873 11.4296 3.31826 11.1697 3.5Z"
                        fill={COLORS.default}
                    />
                </svg>
            ),`

If these are too many changes I can fix them and make a new PR